src/app/about/page.tsx

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+export default function About() {
+  return (
+    <>
+      <Link href="/">
+        <button className="btn">X</button>
```

Comment:
I won't suggest using a `button` inside a `link`. 
If the `btn` class does not keep the desired styling for a `div` element, then leave it as it is. If not, I would suggest using a `div` instead. 

Also the `href="/"` is intended or a simply placeholder until you refactor it?

Reply:
The button was just a provisionally way to get back to the home page. 

The `href="/"` was intended, it directs you to the root folder where I have my homepage as: page.tsx.